i18n/locales/en.ts

```diff
@@ -2737,49 +2737,52 @@ Stack Hint: To spend before the secret is revealed, Vanderpoole uses his signatu
       nav_title: 'The off-chain transaction',
       heading_one: 'The off-chain transaction',
       paragraph_one:
-        "Let's create a valid Bitcoin transaction but instead of sending it to the network, we'll just send it directly to Laszlo! Laszlo should trade you a beer for this transaction because he knows he effectively has the money even though the transaction is not broadcasted or confirmed.",
-      paragraph_two:
-        "Remember it's the year 2140! A SHA-256 Stout cost 0.00010000 BTC.",
-      heading_two: 'Hints',
+        "Let's create a valid bitcoin transaction but instead of broadcasting it out to the network, we'll give it directly to Laszlo. In return, he should provide you with a beer because he can broadcast the transaction anytime he wants. Once it's confirmed and mined into a block, the money is his.",
+      paragraph_two: `Remember it's the year 2140! A <span className="italic">SHA-256 Stout</span> costs 0.00010000 BTC.`,
```

Comment:
should be `0.00001000 BTC` I think -- 1,000 sats?

Reply:
Good catch! Fixed and force pushed [caf365a](https://github.com/saving-satoshi/saving-satoshi/pull/1174/commits/caf365a783a8215082e48cd4319f2aa1ae5fe854)